providers/hubspot/read.go

```diff
@@ -105,3 +94,32 @@ func makeQueryValues(config common.ReadParams) []string {
 
 	return out
 }
+
+func buildSearchParams(config common.ReadParams) SearchParams {
+	searchParams := SearchParams{
+		ObjectName: config.ObjectName,
+		SortBy: []SortBy{
+			BuildSort(ObjectFieldHsObjectId, SortDirectionAsc),
+		},
+		NextPage: config.NextPage,
+		Fields:   config.Fields,
```

Comment:
maybe i'm missing something but I don't actually see the code where you're replacing the fields list with `includeAllProperties`

Reply:
Added this in the description - ```Hubspot has a includeAllProperties param, but it hasn't been implemented for all endpoints. The easier fix is to route potentially very long queries to the search endpoint where we list properties in the request body.
```

For search, we don't need to change anything since the request body can handle as many fields as required